File: thorium/src/test/benchmark/add-test/testAdd.R
csv <- "./src/test/benchmark/add-test/data.csv"
table <- read.csv(csv, stringsAsFactors = FALSE)
table[nrow(table) + 1,] <- c("pierre",21," rennes","GMA")
rows <- list(c("serge",21,"rennes","GMA"),c("paul",22," paris","GMA"),c("herve",23," lyon","INFO"))
for (row in rows){
	table[nrow(table) + 1,] <- row
}
write.csv(table, "./src/test/benchmark/add-test/R-add.csv", row.names=FALSE, quote=FALSE)

File: thorium/src/test/benchmark/add-test/testAdd.py
import pandas as pd
csv = "./src/test/benchmark/add-test/data.csv"
table = pd.read_csv(csv)
values = "pierre,21, rennes,GMA".split(",")
new_row = pd.DataFrame([values],columns=table.columns)
table = pd.concat([table,new_row], ignore_index=True)
new_values = ["serge,21,rennes,GMA","paul,22, paris,GMA","herve,23, lyon,INFO"]
for row in new_values:
	values = row.split(",")
	new_row = pd.DataFrame([values],columns=table.columns)
	table = pd.concat([table,new_row], ignore_index=True)
table.to_csv("./src/test/benchmark/add-test/Python-add.csv",index=False)
